### PR TITLE
[Tweak] Pen are wearable on ear + cleaning

### PR DESCRIPTION
**Pull request by lzk228**
## About the PR

requires #18384 

**Media**

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- tweak: Spessmens learned to put the pen behind their ear.

### CODE REVIEW

**Comment by UbaserB** [comment]
I have a questions. Are other pens like cybersun pens and explosive/hypo allowed to be placed on the ear?

File: Resources/Prototypes/Entities/Objects/Misc/paper.yml [hunk 1]
```diff
@@ -224,10 +224,14 @@
   - type: PhysicalComposition
     materialComposition:
       Steel: 25
+  - type: Clothing
+    slots:
+    - earl
+    - earr
```
Comment (ficcialfaint) [inline Resources/Prototypes/Entities/Objects/Misc/paper.yml:230]:
```suggestion
    slots:
    - ears
```

**Review by ficcialfaint — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Misc/paper.yml (hunk 1 above)
Comment (lzk228) [reply Resources/Prototypes/Entities/Objects/Misc/paper.yml:230]: read pr desc pls

Comment (ficcialfaint) [reply Resources/Prototypes/Entities/Objects/Misc/paper.yml:230]: eh aight

**Pull request closed without merging**